docs/fs.md

```diff
@@ -31,9 +35,14 @@ _Run virtiofsd_
     --cache=never
 ```
 
-The `cache=never` option should be the default when using `virtiofsd` with the __cloud-hypervisor__ VMM. This prevents from using the guest page cache, which reduces the memory footprint of the guest. When running multiple virtual machines on the same host, this will let the host deal with page cache, which will increase the density of virtual machines which can be launched.
+The `cache=never` option should be the default when using `virtiofsd` with the
```

Comment:
"Should be"? Would it not be better to just state it is the default and its behaviour.